src/ukify/ukify.py

```diff
@@ -1587,6 +1708,14 @@ def __call__(
         config_key='UKI/DeviceTreeAuto',
         config_push=ConfigItem.config_list_prepend,
     ),
+    ConfigItem(
+        '--hwids-directory',
```

Comment:
I don't think we should have `-directory` in the name. Just say
`--hwids`, `metavar='DIR'`.